src/Avalonia.Controls/Presenters/TextPresenter.cs

```diff
@@ -61,11 +92,104 @@ static TextPresenter()
 
         public TextPresenter()
         {
-            _caretTimer = new DispatcherTimer();
-            _caretTimer.Interval = TimeSpan.FromMilliseconds(500);
+            _text = string.Empty;
+            _caretTimer = new DispatcherTimer { Interval = TimeSpan.FromMilliseconds(500) };
             _caretTimer.Tick += CaretTimerTick;
         }
 
+        /// <summary>
+        /// Gets or sets a brush used to paint the control's background.
+        /// </summary>
+        public IBrush Background
```

Comment:
Nit: Don't we usually use expression bodied properties in new code?

Reply:
This is just a copy paste because TextPresenter no longer inherits from TextBlock. If you insist on this change I am fine with changing to expression bodies.

Reply:
I don’t really mind, but I usually convert to expression bodies. This is just a nitpick so ultimately it is up to you.